x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/cache/FrozenCacheService.java

```diff
@@ -604,101 +603,33 @@ public boolean isEvicted() {
             return evicted.get();
         }
 
-        public boolean isReleased() {
-            return isEvicted() && refCount() == 0;
-        }
-
         @Override
         protected void closeInternal() {
             // now actually free the region associated with this chunk
             onClose(this);
             logger.trace("closed {} with channel offset {}", regionKey, physicalStartOffset());
         }
 
-        private void ensureOpen() {
```

Comment:
why did you remove this? This helps with avoiding to hold onto cache regions when they are supposed to be evicted (i.e. allows writers/readers to quickly fail)